views/download.html.twig

```diff
@@ -1,64 +1,120 @@
 {% extends "layout.html.twig" %}
 
 {% block content %}
-    <h1>Download Composer <span class="latest">Latest: v{{ latestStable }}</span></h1>
 
-    {% if windows %}
-        <h2>Windows Installer</h2>
-        <p>The installer will download composer for you and set up your PATH environment variable so you can simply call <code>composer</code> from any directory.</p>
-        <p>Download and run <a href="/Composer-Setup.exe">Composer-Setup.exe</a> - it will install the latest composer version whenever it is executed.</p>
+    <section role="main">
```

Comment:
Should this not just be `<main>` then?

Reply:
There's already a <main> in the layout, this is the semantics for the main <section> (ARIA role https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques).

Reply:
I'll replace with `<article role="article">` which makes more sens, considering this is the contents of this page.

Reply:
Closed: changed the HTML markup accordingly.